apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -226,6 +231,10 @@ function initializeAccelerometer(board) {
     board,
     controller: PlaygroundIO.Accelerometer
   });
+
+  accelerometer.start = function () {
+    accelerometer.io.sysexCommand([CP_COMMAND, CP_ACCEL_STREAM_ON]);
+  };
```

Comment:
Restoring the `start()` function.  See [how playground-io enables accelerometer streaming when an event handler is attached](https://github.com/rwaldron/playground-io/blob/master/lib/index.js#L158-L165).  I confirmed that it's a no-op to send this command more than once.

Practical upshot is that you _don't_ actually need to call this if you're using accelerometer events, but if you just want to read `accelerometer.x` or something you'll need to call `start()` first.